src/main/java/de/mossgrabers/convertwithmoss/format/yamaha/ysfc/YamahaYsfcCreator.java

```diff
@@ -173,11 +174,29 @@ public void create (final File destinationFolder, final List<IMultisampleSource>
      */
     private static void storeMultisamples (final List<IMultisampleSource> multisampleSources, final File multiFile, final OutputFormat outputFormat) throws IOException
     {
-        final YsfcFile ysfcFile = new YsfcFile ();
+        final YsfcFile ysfcFile;
+        if (!outputFormat.equals(OutputFormat.MOXF_LIBRARY)) {
+            ysfcFile = YsfcFile.withChunks ("EWFM", "DWFM", "EWIM", "DWIM");
+        } else {
+            ysfcFile = YsfcFile.withChunks ("EWFM", "DWFM", "EWIM", "DWIM", "EARP", "DARP", "EVCE", "DVCE");
```

Comment:
I was not sure if the chunks in X3A file can be skipped, so added all of them and left them empty.